src/authentication/MobileRouter.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```

Comment:
This is the main component of the module, the "public API" so to speak. It takes the app's "normal" routes and information about the user's current state (`isAuthenticated `, `isRevoked`, etc) ans toggles between different views.